anomaly-benchmarking/generate-datasets/script.whizzml

```diff
@@ -21,27 +30,37 @@
 ;; returns a dataset of only anomalous points and a new field
 ;; labeling each row by one of four difficulty levels.
 
-(define (point-difficulty ds-id new-obj delete-resources)
+(define (point-difficulty ds-id new-obj delete-resources diff-list)
   (let (ds (all-preferred ds-id)
         lr-id (create-and-wait-logisticregression ds)
-        bp-id (wait (create-batchprediction lr-id ds {"probabilities" true 
-                                                      "output_dataset" true 
-                                                      "all_fields" true }))
+        bp-id (wait (create-batchprediction lr-id 
+                                            ds 
+                                            {"probabilities" true 
+                                             "output_dataset" true 
+                                             "all_fields" true 
+                                             "prediction_name" "bp"}))
```

Comment:
this will break if an objective field is named "bp", won't it? you could do something like defining `pred-name (str "prediction for " ((fetch ds) ["objective_field" "name"]))` and then use `pred-name` here and below in the `get-id` call.